Tasks/CodeDeployDeployApplication/TaskOperations.ts

```diff
@@ -0,0 +1,269 @@
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we need to change the copyright header to 2019?

Reply:
I talked about this with Paul when I started refactoring and it seems the answer is no